buf.gen.yaml

```diff
@@ -8,6 +8,24 @@ managed:
     except:
       - buf.build/googleapis/googleapis
 plugins:
+  - remote: buf.build/protocolbuffers/plugins/go:v1.27.1-1
+    out: proto/
+    opt:
+      - paths=source_relative
+  - remote: buf.build/grpc/plugins/go:v1.1.0-1
+    out: proto/
+    opt:
+      - paths=source_relative
+  - remote: buf.build/jirkad/plugins/protoc-gen-validate:v0.6.1
```

Comment:
Let me see what we're using in our generation template in the Buf Remote Registry. We'll want to use the same versions here and there so that the generated sources are identical.

Reply:
I updated a few of them to match the ones we use in the Buf Remote Registry template.
https://buf.build/openfga/templates/go

👍